static/src/stylesheets/amp/_hosted.scss

```diff
@@ -11,7 +11,7 @@ $header-height: 48px;
     text-align: right;
 }
 
-.hosted__glogo svg {
+.hosted__glogo .inline-logo__svg {
```

Comment:
I still think not nesting would be the most optimal for this. If you add any extra class to this svg, it will automatically add that class with `__svg` after it. But definitely better! 😄